tests/10apidoc/33room-members.pl

```diff
@@ -324,6 +336,10 @@ sub matrix_create_and_join_room
 
       log_if_fail "room_id=$room_id";
 
+      ( $options{with_invite} ?
```

Comment:
FYI: strictly this isn't actually necessary, because a `Future->needs_all` over zero components is just immediately true with no results, but it keeps a neat consistency with the other options, so the code can stay as it is.